src/lido/ids.ts

```diff
@@ -0,0 +1,5 @@
+import { PublicKey } from "@solana/web3.js";
+
+export const LIDO_PROGRAM_ID = new PublicKey("CrX7kMhLC3cSsXJdT7JDgqrRVWGnUpX3gfEfxxU2NVLi");
+export const LIDO_ADDRESS = new PublicKey("49Yi1TKkNyYjPAFdR9LBvoHcUjuPX4Df5T5yv39w2XTn");
+export const ST_SOL_MINT_ADDRESS = new PublicKey("7dHbWXmci3dT8UFYWYZweBLXgycu7Y3iL6trKn1Y7ARj");
```

Comment:
You can import them from our [sdk](https://github.com/lidofinance/solido-sdk/blob/6fadb02977302f73d17bfdaa74ca1479066cd584/src/constants.ts#L17), why not to use it?